go/ethadapter/erc20contractlib/erc20_contract_lib.go

```diff
@@ -54,7 +54,7 @@ func NewERC20ContractLib(mgmtContractAddr *gethcommon.Address, contractAddrs ...
 }
 
 func (c *erc20ContractLibImpl) CreateDepositTx(tx *ethadapter.L1DepositTx, nonce uint64) types.TxData {
-	data, err := c.contractABI.Pack("transfer", &tx.To, big.NewInt(int64(tx.Amount)))
+	data, err := c.contractABI.Pack("transfer", &tx.To, tx.Amount)
```

Comment:
It's fine to pack big.Int, yes ?

Reply:
More than fine, it's required haha. See how before the int64 value was getting converted to big.Int for packing